File: 0013-roman-to-integer/0013-roman-to-integer.c
int romanToInt(char* s) {
    int num=0;
    for(int i=0;i<strlen(s);i++){
        printf("%d ",s[i]);
        if(s[i]=='I'){
            if(s[i+1]=='V'){
                num+=4;
                i++;
            }
            else if(s[i+1]=='X'){
                num+=9;
                i++;
            }
            else{
                num+=1;
            }
        }
        else if(s[i]=='X'){
            if(s[i+1]=='L'){
                num+=40;
                i++;
            }
            else if(s[i+1]=='C'){
                num+=90;
                i++;
            }
            else
                num+=10;
        }
        else if(s[i]=='C'){
            if(s[i+1]=='D'){
                num+=400;
                i++;
            }
            else if(s[i+1]=='M'){
                num+=900;
                i++;
            }
            else
                num+=100;
        }
        else if(s[i]=='V'){
            num+=5;
        }
        else if(s[i]=='L'){
            num+=50;
        }
        else if(s[i]=='D'){
            num+=500;
        }
        else if(s[i]=='M'){
            num+=1000;
        }
    }
    printf("\n%d",num);
    return num;
}

File: 0013-roman-to-integer/0013-roman-to-integer.py
class Solution(object):
    def romanToInt(self, s):
        num=0
        i=0
        d={'I':1,'V':5,'X':10,'L':50,'C':100,'D':500,'M':1000}
        while i<len(s):
            print i
            if s[i]=='I' and i<len(s)-1:
                if s[i+1]=='V':
                    i=i+2
                    num=num+4
                    continue
                elif s[i+1]=='X':
                    i=i+2
                    num=num+9
                    continue
            elif s[i]=='X'and i<len(s)-1:
                if s[i+1]=='L':
                    i=i+2
                    num=num+40
                    continue
                elif s[i+1]=='C':
                    i=i+2
                    num=num+90
                    continue
            elif s[i]=='C'and i<len(s)-1:
                if s[i+1]=='D':
                    i=i+2
                    num=num+400
                    continue
                elif s[i+1]=='M':
                    i=i+2
                    num=num+900
                    continue
            num+=d[s[i]]
            i+=1
            #num+=d[s[i]]
            #i+=1
        return num